readthedocs/templates/projects/project_version_list.html

```diff
@@ -104,7 +119,7 @@ <h1>{% trans "Inactive Versions" %}</h1>
 
                       {% if request.user|is_admin:project %}
                         <ul class="module-item-menu">
-                          <li><a href="{% url "project_version_detail" project.slug version.slug %}">{% trans "Edit" %}</a></li>
+                          <li><a href="{% url "project_version_detail" project.slug version.slug %}">{% trans "Edit & Activate" %}</a></li>
```

Comment:
I don't like the "Edit & Activate" label for the button, but I do not have a better one. Do you think that just "Edit" as it was does not communicate it properly?

Reply:
I might have the same feeling. The phrase sounds overly specific for a user. Technically, from our perspective, they are editing and activating, but to the user, they are just activating. I think "Activate" is the most correct descriptor for what the user is looking to accomplish, we can gloss over the fact that it drops into a form.

Reply:
Sounds good. I'll update it.